content/docs/start/index.md

```diff
@@ -52,14 +52,14 @@ trail below to see an overview of all features from that perspective:
 ### Data Management
 
 - **[Data and model versioning]** is the base layer of DVC for large files,
-  datasets, and machine learning models. Use a regular Git workflow, but without
-  storing large files in the repo. Data is cached by DVC, allowing for efficient
-  sharing. Think "Git for data".
-
-- **[Data and model access]** shows how to bring, explore, and access data
-  artifacts from outside the project. This can help download a specific version
-  of an ML model to a deployment server or import a dataset to another project,
-  for example.
+  datasets, and machine learning models. Use a standard Git workflow, but
+  without storing large files in the repo. Data is cached by DVC, allowing for
+  efficient sharing. Think "Git for data".
+
+- **[Data and model access]** covers using data artifacts from outside of the
+  project and/or importing data artifacts from another DVC project. This can
+  help to download a specific version of an ML model to a deployment server or
+  import a model to another project, for example.
```

Comment:
Wait. I'm not sure I meant to change this one... Reverting (mostly)

```suggestion
- **[Data and model access]** shows how to bring, explore, and access data
  artifacts from outside the project. This can help download a specific version
  of an ML model to a deployment server or import a dataset tino another
  project, for example.
```